MiniGames/MiniGames/round_result.cpp

```diff
@@ -0,0 +1,41 @@
+// round_result.cpp -- program for determining the winner
+
+
+int get_choice_from_symbol(char);
```

Comment:
Remove this forward declaration, and just move definition here so it's visible in "round_result" function